ui/src/dashboards/actions/thunks.ts

```diff
@@ -286,7 +286,7 @@ export const getDashboard = (dashboardID: string) => async (
       throw new Error(resp.data.message)
     }
 
-    dispatch(hydrateVariables())
+    dispatch(hydrateVariables(true))
```

Comment:
```suggestion
    const skipCache = true
    dispatch(hydrateVariables(skipCache))
```

Reply:
is the request to be more explicit about what it's doing through the var name?

Reply:
yup :)

Reply:
💯